demo/src/main/java/com/novoda/movies/rateable/RateableMoviesActivity.java

```diff
@@ -0,0 +1,65 @@
+package com.novoda.movies.rateable;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.widget.Toast;
+
+import com.novoda.movies.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RateableMoviesActivity extends AppCompatActivity {
```

Comment:
the demo is trying not to focus on anything except the viewholder. obviously in your production code you won't have something called "FakeMovieRepository" that creates a list of movies from a static method.